common/src/main/java/com/wynntils/models/containers/type/SearchableContainerType.java

```diff
@@ -5,33 +5,59 @@
 package com.wynntils.models.containers.type;
 
 import com.wynntils.core.text.StyledText;
+import com.wynntils.utils.wynn.ContainerUtils;
+import java.util.function.Predicate;
 import java.util.regex.Pattern;
+import java.util.stream.Stream;
+import net.minecraft.world.inventory.ChestMenu;
+import net.minecraft.world.item.ItemStack;
 
 public enum SearchableContainerType {
-    BANK(Pattern.compile("§0\\[Pg. (\\d+)\\] §8(.*)'s?§0 Bank"), Pattern.compile("§f§lPage \\d+§a >§2>§a>§2>§a>"), 8),
+    BANK(
+            Pattern.compile("§0\\[Pg. (\\d+)\\] §8(.*)'s?§0 Bank"),
+            Pattern.compile("§f§lPage \\d+§a >§2>§a>§2>§a>"),
+            8,
+            slotPos -> slotPos.x() < 7 && slotPos.y() < 6),
```

Comment:
It looks like the test methods are very similar. You can extract them to two helper methods, like `testBankStyleSlot` and `testGuildStyleSlot`.